be/src/exec/tablet_sink.cpp

```diff
@@ -785,9 +786,14 @@ int OlapTableSink::_validate_data(RuntimeState* state, RowBatch* batch, Bitmap*
     for (int row_no = 0; row_no < batch->num_rows(); ++row_no) {
         Tuple* tuple = batch->get_row(row_no)->get_tuple(0);
         bool row_valid = true;
+        std::stringstream ss; // error message
         for (int i = 0; row_valid && i < _output_tuple_desc->slots().size(); ++i) {
             SlotDescriptor* desc = _output_tuple_desc->slots()[i];
-            if (desc->is_nullable() && tuple->is_null(desc->null_indicator_offset())) {
+            if (tuple->is_null(desc->null_indicator_offset())) {
```

Comment:
Why remove `desc->is_nullable()`.
If desc is not nullable, the `desc->null_indicator_offset()` is meaningless.
It will lead error.

Reply:
oops, you're right. I'll fix this.